content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,462 @@
+
+All pickers have been reworked for a global behaviour and a more homogeneous display throughout the interface. This work also adds functionality for resizing the areas defined by a picker.
+
+The file selector interface has also been redesigned.
+
+The export and print progress bar has been visually redesigned for finer integration with the interface.
+
+In _darkroom_, the active module is now shown with a new visual effect on status icon and module title : you could see that on _filmic v4_ screenshot, in the presentation of this new version, further down in the article.
+
+The list of location search results in map view finally looks good and more readable.
+
+All the sliders have been refined: the bars are thinner and the control sliders more visible.
+
+And a few other little details all over the place.
+
+# Digital Asset Management
+
+## Metadata editor and tagging evolve
+
+The developments are not huge but more than welcome.
+
+### Metadata editor
+
+Until now, the editor had five fields, now it has two more: a "notes" field, to add notes ; and a "version name" to, for example, customize the name of different versions of the same image. These new fields have been, of course, integrated to import options. These import options have been too, harmonized between the three different ways to import images (and now each field could be activated or not on import).
+
+All editor fields are now multi-lines. Simply press `Ctrl` + `enter` to add a line in a field (pressing `enter` only makes you go to the next field by saving the previously completed one). Thus, fields can also be expanded with `Ctrl` + `scroll`.
+
+By opening this new editor, you will also see a new setting icon. For each editor field, you can choose to hide it from the editor, or to make it private (not exportable).
+
+### Tagging
+
+darktable 3.2 further improves tag management. On darktable 3.0.2, it was not always possible to add a tag on an already existing parent. It is now possible systematically with the improvement of virtual node (not used tag) management. These last ones are now shown in italic.
+
+Tag creation now works without a selected image. It is now possible to create a tag on a virtual node, insert a pipe character | in the creation tag. The tree view shows the newly created tags.
+
+But most importantly, an important cleanup has been done, solving some bugs. For example, updating a tag could required restarting darktable to be taken into account; this is no longer necessary. In short, darktable 3.0 initiated a new keyword module, darktable 3.2 refines and finalizes it.
+
+## New collection filters
```

Comment:
Not sure this is the right place ...
it should be said that when "tag" is the first level collection, any change about the images order is saved along with the selected tag. This allows to associate a specific order, even the custom one, to every (tag) collection of images.

Reply:
I'm not sure it is so important there. Nothing wrong here anyway, even if it's not so precise. The article is already ready in 3 languages and has been reviewed by quite some different people. It's just an example and the most important thing is that some bugs had been solved.